src/App.css

```diff
@@ -1,4 +1,283 @@
-.author-wrapper {
-  background-color: white;
-  margin-left: 2%;
+:root {
+  --border-color: #f1f1f1;
+  --backround-color: #fafafa;
+  --primary-color: #0290ff;
+  --accent-color: #8a8a8a;
+  --max-width: 1000px;
+}
+
+.nav-container {
+  display: grid;
+  place-items: center;
+  border-bottom: 2px solid var(--border-color);
+  margin-bottom: 2%;
+  margin-left: -6.3rem;
+}
+
+.nav-wrapper {
+  max-width: var(--max-width);
+  width: 80%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.left-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 30px;
+}
+
+.nav-wrapper .nav-title {
+  color: var(--primary-color);
+  font-weight: 800;
+  font-size: 40px;
+  padding-block: 1.4rem;
+}
+
+.link-container {
+  display: flex;
+  gap: 10px;
+}
+
+.link-container a {
+  text-decoration: none;
+  font-size: 20px;
+  letter-spacing: 2px;
+  color: #121212;
+}
+
+.link-container a.active {
+  font-weight: 700;
+}
+
+.page {
+  background-color: var(--backround-color);
+}
+
+.book-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  padding-top: 50px;
+}
+
+.book-card {
+  max-width: var(--max-width);
+  width: 90%;
+  box-sizing: border-box;
+  border: 1px solid var(--border-color);
+  padding: 30px;
+  background-color: #fff;
+  display: flex;
+  gap: 30px;
+  margin-left: 8rem;
+  margin-bottom: 1.5%;
+}
+
+.book-card-left-container {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+}
+
+.book-card-right-container {
+  width: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+}
+
+.book-card-button-container {
+  display: flex;
+  align-items: center;
+  margin-top: 8%;
+  gap: 5px;
+}
+
+.book-card-button-container button {
+  background-color: transparent;
+  border: none;
+  font-weight: 200;
+  padding: 0;
+  width: 4.563rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 0.875rem;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.vertical-divider {
+  border-left: 2px solid var(--primary-color);
+  height: 80%;
+  border-left-color: #afccdb;
+  margin: 0;
+}
+
+.divider-book-form {
+  border-top: 1px solid var(--border-color);
+  max-width: 1000px;
+  width: 80%;
+}
+
+.book-category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 1.3rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title {
```

Comment:
- [ ] wonderful job so far. please, like the previous reviewer  noted, this milestone is aimed at our styling looking almost like the zeplin design. Kindly align the book cards with the page title and user logo placeholder like in the original design. 
yours
![Screenshot (1267)_LI](https://github.com/rbd3/bookstore/assets/65298048/e9dfc4f5-d61d-428d-a896-f6c698d46318)
original
![Screenshot (1266)](https://github.com/rbd3/bookstore/assets/65298048/c5d6213f-2d4b-49f4-b28d-2bd153f6a759)